msal-dotnet-articles/contribute/prerequisites-to-build.md

```diff
@@ -1,26 +1,23 @@
 ---
 title: Prerequisites to build MSAL.NET
+description: "Requirements to build MSAL.NET on your local computer."
 ---
 
 # Prerequisites to build MSAL.NET
 
-## Components required to build the library
-
-The following are instructions to setup Visual Studio (Mac) to build various solution files on Windows and Mac platform.
+The following are instructions to setup Visual Studio to build various MSAL.NET solution files on Windows and Mac platforms.
 
 ## Windows
 
-### Minimal Visual Studio 2019 installation
-
-* Install or update Visual Studio 2019 (any edition: Community, Pro, or Enterprise) with the following workloads:
+### Minimal Visual Studio installation
 
+* Install or update Visual Studio 2022. Any edition, such as Community, Pro, or Enterprise will work.
+* Install the following workloads:
   * .NET desktop development
   * Universal Windows Platform development
   * Mobile Development with .NET
   * .NET Core cross-platform development
-
-* Then from the "Individual Components" tab,make sure these additional items are selected:
-
+* From the **Individual Components** tab, make sure these items are selected:
   * .NET Framework 4.5 targeting pack
```

Comment:
It does no longer exist. Where did you find it, @localden?

Reply:
Good catch, @jmprieur - this should be 4.5.2.
![image](https://github.com/MicrosoftDocs/microsoft-authentication-library-dotnet/assets/53200638/efe28b43-b88d-46ff-8def-64178e7ddee6)